edi_ubl_oca/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Camptcamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "EDI UBL",
+    "summary": """Define EDI backend type for UBL.""",
+    "version": "14.0.1.0.0",
+    "development_status": "Alpha",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/edi",
+    "author": "Camptcamp,Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "Camptocamp,Odoo Community Association (OCA)",
```
:)

Reply:
@simahawk There are many typo like this, please review them all

Reply:
good catch, I fixed in other modules in this repo https://github.com/OCA/edi/pull/514 I'll do it here too. Tnx!